go/systests/teams_test.go

```diff
@@ -1661,7 +1661,7 @@ func TestBatchAddMembersCLI(t *testing.T) {
 		{AssertionOrEmail: botua.username, Role: keybase1.TeamRole_BOT},
 		{AssertionOrEmail: restrictedBotua.username, Role: keybase1.TeamRole_RESTRICTEDBOT, BotSettings: &keybase1.TeamBotSettings{}},
 	}
-	added, notAdded, err := teams.AddMembers(context.Background(), alice.tc.G, teamID, users)
+	added, notAdded, err := teams.AddMembers(context.Background(), alice.tc.G, teamID, users, nil)
```

Comment:
usually we try to comment and name boolean and `nil` values in function calls like so:
```
added, notAdded, err := teams.AddMembers(context.Background(), alice.tc.G, teamID, users, nil /* emailInviteMsg */)
```
since this is test code, it probably matters less, but is still nice for anyone who looks at this in the future

Reply:
Gotcha, I'll add that